meshroom/nodes/aliceVision/CameraInit.py

```diff
@@ -163,6 +163,27 @@ class CameraInit(desc.CommandLineNode, desc.InitNode):
             value='${ALICEVISION_SENSOR_DB}',
             uid=[],
         ),
+        desc.File(
+            name='lensCorrectionProfileInfo',
+            label='Lens Correction Profile Info',
+            description='''Lens Correction Profile filepath or database directory.''',
+            value='${ALICEVISION_LENS_PROFILE_INFO}',
+            uid=[],
+        ),
+        desc.BoolParam(
+            name='enableLensCorrectionProfileSearch',
+            label='Enable Automatic LCP Search',
+            description='''Enable automatic LCP Search if a database directory is specified as LCP info.''',
+            value=False,
+            uid=[0],
+        ),
+        desc.BoolParam(
+            name='lensCorrectionProfileSearchByLensNameAndCameraMakerOnly',
```

Comment:
This parameter name doesn't correspond to any argument of the cameraInit command-line tool, which is causing an error whenever we try to import images

Reply:
Same for `useInternalWhiteBalance`